emukit/examples/models/random_forest.py

```diff
@@ -3,64 +3,43 @@
 
 import numpy as np
 
-from ..core.interfaces.models import IModel
+from ...core.interfaces.models import IModel
 
 try:
-    import pyrfr.regression as reg
+    from sklearn.ensemble import RandomForestRegressor
 except ImportError:
     raise ImportError("""
-        This module is missing required dependencies. Try running
+        This module is missing required dependencies. Try running:
 
-        pip install git+https://github.com/automl/random_forest_run.git
+        pip install scikit-learn
+
+        Refer to http://scikit-learn.org/stable/install.html for further information.
     """)
 
 
 class RandomForest(IModel):
 
     def __init__(self, X_init: np.ndarray, Y_init: np.ndarray, num_trees: int = 30,
```

Comment:
how do you feel about allowing the user to pass in kwargs for the sklearn model? We don't want to expose all options in the sklearn random forest constructor but we are limiting what a user can do here.

Reply:
Personally, I feel negatively, but I can be persuaded... 

Anyways, the sole point of existence of these models, from my point of view, was to give users something simple to play with out of the box. They will be limiting no doubt, but then they are not intended for a real modelling.

Reply:
But if they can be made ready for "real" modelling with such a small change, shouldn't we do it? I think it would be quite frustrating if I had to re-implement this class just so I can pass in one more parameter to the sklearn random forest constructor.

An alternative might be to have a "SciKitLearnRandomForestWrapper" and simply have a function that creates a random forest with default parameters, passes it in to the wrapper then returns this wrapper to the user. Then if a user wants to create a custom random forest model they can reuse the wrapper.